x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings/latest_findings_container.tsx

```diff
@@ -81,9 +83,12 @@ export const LatestFindingsContainer = ({ dataView }: FindingsBaseProps) => {
         {!error && (
           <>
             <FindingsGroupBySelector type="default" />
-            {findingsGroupByNone.isSuccess && (
+            {findingsGroupByNone.isSuccess && !!findingsGroupByNone.data.page.length && (
               <FindingsDistributionBar
                 {...{
+                  type: i18n.translate('xpack.csp.latestFindingsTableHeader.tableRowTypeLabel', {
```

Comment:
This ID feels wrong (missing a `.findings` section for instance) please make sure it is algined with @JordanSh work regarding i18n IDs in this page